File: CyFile-Android/app/src/main/java/at/tugraz/tc/cyfile/logging/impl/AndroidLogger.java
package at.tugraz.tc.cyfile.logging.impl;

import android.util.Log;

import at.tugraz.tc.cyfile.logging.CyFileLogger;

public class AndroidLogger implements CyFileLogger {

    @Override
    public void d(String tag, String msg) {
        Log.d(tag, msg);
    }

    @Override
    public void e(String tag, String msg) {
        Log.e(tag, msg);
    }

    @Override
    public void i(String tag, String msg) {
        Log.i(tag, msg);
    }

    @Override
    public void w(String tag, String msg) {
        Log.w(tag, msg);
    }
}

File: CyFile-Android/app/src/main/java/at/tugraz/tc/cyfile/logging/impl/AndroidLogger.kt
package at.tugraz.tc.cyfile.logging.impl

import android.util.Log

import at.tugraz.tc.cyfile.logging.CyFileLogger

class AndroidLogger : CyFileLogger {

    override fun d(tag: String, msg: String) {
        Log.d(tag, msg)
    }

    override fun e(tag: String, msg: String) {
        Log.e(tag, msg)
    }

    override fun i(tag: String, msg: String) {
        Log.i(tag, msg)
    }

    override fun w(tag: String, msg: String) {
        Log.w(tag, msg)
    }
}
